### PR TITLE
Address spelling mistake in specification

### PR DESCRIPTION
**Pull request by michaelsw95**
"heterogenous" seems to be a medical/biological term. I think this is what was intended: "heterogeneous"

### CODE REVIEW

File: _format/1.0/index.md [hunk 1]
```diff
@@ -1560,7 +1560,7 @@ is an empty array or an array of [resource identifier objects][resource identifi
 If a client makes a `PATCH` request to a URL from a to-many
 [relationship link][relationships], the server **MUST** either completely
 replace every member of the relationship, return an appropriate error response
-if some resources can not be found or accessed, or return a `403 Forbidden`
+if some resources cannot be found or accessed, or return a `403 Forbidden`
```
Comment (dgeb) [inline _format/1.0/index.md]: After reviewing a few sources, it appears that `can not` is just as acceptable as `cannot`. Is there a reason you made this change?

**Comment by dgeb** [comment]
Thanks for these corrections. As noted inline, my only concern is regarding "can not" vs. "cannot", and my preference to avoid making unnecessary changes.

File: _format/1.0/index.md (hunk 1 above)
Comment (michaelsw95) [reply _format/1.0/index.md]: Agreed, my mistake. I have reverted this.

File: _format/1.0/index.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ A document **MUST** contain at least one of the following top-level members:
 * `meta`: a [meta object][meta] that contains non-standard
   meta-information.
 
-The members `data` and `errors` **MUST NOT** coexist in the same document.
+The member's `data` and `errors` **MUST NOT** coexist in the same document.
```
Comment (tobyzerner) [inline _format/1.0/index.md]: This change is incorrect. The `data` and `errors` do not belong to "the member", they ARE members (plural) themselves.

**Comment by tobyzerner** [comment]
For reference re: heterogenous vs heterogeneous
https://grammarist.com/usage/heterogeneous-heterogenous/

File: _format/1.0/index.md (hunk 2 above)
Comment (michaelsw95) [reply _format/1.0/index.md]: My mistake, I have reverted this change

**Pull request merged**